privatebin/privatebin.dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM janitortechnology/ubuntu-dev
+MAINTAINER PrivateBin <support@privatebin.org>
+
+# Download deps
+RUN sudo apt-get update && \
+    sudo apt-get -y install --no-install-recommends nginx php-fpm php-gd php-sqlite3 phpunit && \
+    npm install -g mocha jsverify jsdom@9 jsdom-global@2 mime-types && \
+
+# Configure php-fpm
+    sudo mkdir -p /run/php && \
+    sudo sed -ri "s/^;daemonize = .*$/daemonize = no/" /etc/php/7.0/fpm/php-fpm.conf && \
+    sudo sed -ri "s/^(user|group) = .*$/\1 = user/" /etc/php/7.0/fpm/pool.d/www.conf && \
```

Comment:
Nit: Oh, are you sure that the multi-line `RUN` trick with `&& \` works when you insert empty lines and comments below? We should check if the CI Docker build complained about this.

Reply:
I have only tested this on Docker 1.13.1 (the docker.io package from the Ubuntu 16.04 repos). If I switch to the method outlined by you below it will certainly be more robust.